docs/guides/composition-revisions.md

```diff
@@ -7,8 +7,8 @@ back changes to a Crossplane [`Composition`][composition-type]. It assumes
 familiarity with Crossplane, and particularly with
 [Composition][composition-term].
 
-> Composition Revisions are an __alpha feature__. They are not yet recommended
-> for production use, and are disabled by default.
+> Composition Revisions are a __beta feature__. They are not yet recommended
+> for production use, and are enabled by default.
```

Comment:
```suggestion
> Composition Revisions are a __beta feature__. They are not yet recommended
> for production use, but are enabled by default.
```

I could also be convinced to remove or soften this "not for production use" wording.